File: wm/bbkeys09/patches/patch-src_Config.cpp
$NetBSD: patch-src_Config.cpp,v 1.2 2015/11/17 15:59:13 he Exp $

Include strings.h too.

--- src/Config.cpp.orig	2011-11-25 16:58:41.000000000 +0000
+++ src/Config.cpp
@@ -25,6 +25,7 @@
 
 #include "Config.h"
 #include <string>
+#include <strings.h>
 
 #include <iostream>
 using std::cout;

File: textproc/hs-cgrep/patches/patch-src_Config.hs
$NetBSD: patch-src_Config.hs,v 1.1 2020/01/03 08:22:09 pho Exp $

Compatibility with GHC < 8

--- src/Config.hs.orig	2001-09-09 01:46:40.000000000 +0000
+++ src/Config.hs
@@ -19,6 +19,11 @@
 {-# LANGUAGE DeriveGeneric #-}
 {-# LANGUAGE RecordWildCards #-}
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE CPP #-}
+
+#if !MIN_VERSION_base(4,9,0)
+#  define errorWithoutStackTrace error
+#endif
 
 module Config where
 
